src/storage/http/StorageHttpDownloadHandler.cpp

```diff
@@ -144,46 +190,56 @@ void StorageHttpDownloadHandler::onError(ProxygenError error) noexcept {
                << proxygen::getErrorString(error);
 }
 
-bool StorageHttpDownloadHandler::downloadSSTFiles(const std::string& hdfsHost,
-                                                  int32_t hdfsPort,
-                                                  const std::string& hdfsPath,
-                                                  const std::vector<std::string>& parts) {
-    static std::atomic_flag isRunning = ATOMIC_FLAG_INIT;
-    if (isRunning.test_and_set()) {
```

Comment:
why remove it?

Reply:
concurrently download is needed, with different tag/edge in different sub dirs.

Reply:
Make sense.